src/lang_pawn.cpp

```diff
@@ -275,7 +276,8 @@ static Chunk *pawn_process_variable(Chunk *start)
    LOG_FUNC_ENTRY();
    Chunk *pc = Chunk::NullChunkPtr;
 
-   if (start != nullptr)
+   if (  start != nullptr
+      && start->IsNotNullChunk())
```

Comment:
```if (start != nullptr)``` was added to prevent SEGV during the transition process of the code improvement. At the end this will simply be ```pc = start;``` without any ```if```, because Chunk pointers won't be null.
Checking for ```start->IsNotNullChunk()``` is not wrong, but it is not necessary either. If ```start``` is a null chunk, there is no effect on the current value of ```pc```

Reply:
We can leave the original as is, as explained in previous comment.